cmd/gaia/app/genesis.go

```diff
@@ -114,34 +115,38 @@ func GaiaAppGenState(cdc *codec.Codec, appGenTxs []json.RawMessage) (genesisStat
 			err = errors.New("must provide genesis StdTx with exactly 1 CreateValidator message")
 			return
 		}
-		msg := msgs[0].(stake.MsgCreateValidator)
+		_, ok := msgs[0].(stake.MsgCreateValidator)
+		if !ok {
```

Comment:
```
if _, ok := msgs[0].(stake.MsgCreateValidator); !ok {
```

Reply:
++

Reply:
Done, thanks. I'll open a PR for docs as soon as this is merged